cmd/root.go

```diff
@@ -46,13 +47,16 @@ func Execute() {
 	const threads = 2
 	wg.Add(threads)
 
+	ctx, cancel := context.WithCancel(context.Background())
+
 	go func() {
-		sendAnalytics()
+		sendAnalytics(ctx)
 		wg.Done()
```

Comment:
```suggestion
		defer wg.Done()
		sendAnalytics(ctx)
```

for consistency with the suggestion in lines below. Please accept either both or none.
On a side note, this has a side effect of the `wg.Wait()` call being guaranteed to happen.